sdk/python/feast/type_map.py

```diff
@@ -80,6 +119,10 @@ def python_type_to_feast_value_type(
     if type_name in type_map:
         return type_map[type_name]
 
+    if isinstance(value, list):
```

Comment:
`pd.core.series.Series` accepts a Python as input. The `np.array` cast is redundant here.
we can combine `if isinstance(value, list):` and `if type_name == "ndarray":` into a single if:

```python
if isinstance(value, list) or type_name == "ndarray":
    if recurse:
            # Convert to list type
            list_items = pd.core.series.Series(value)
            ...
```